packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -162,18 +162,16 @@ export class DeltaStorageService implements IDeltaStorageService {
 			async (event) => {
 				const restWrapper = await this.getRestWrapper();
 				const url = this.getDeltaStorageUrl();
-				// If fetchReason is not provided, default to "No reason provided"
-				const reason = fetchReason ?? "No reason provided";
 				const response = await restWrapper.get<ISequencedDocumentMessage[]>(url, {
 					from: from - 1,
 					to,
-					fetchReason: reason,
```

Comment:
Why are you removing it?

Reply:
I removed it from there because I thought that it could break the link if I added it to the response, and I moved it to the telemetry ecause I thought AFR also had access to it, should I change it back?

Reply:
Jatin is correct. AFR wants the fetch reason to be sent across the wire, and the way we do that is through this request.

When making HTTP requests like this, unused query parameters are effectively ignored, so we don't have to worry about breaking anything by adding an extra parameter.
The request URL will look something like `https://url.com/path1/path2?from=0&to=0&fetchReason=reason`